### PR TITLE
additions to css template

### PR DESCRIPTION
**Pull request by alan-y**
Summary of changes:
- All PHS colours defined in main template
- Code in `physhinyapp()` function appends the main css template at the top of the shiny css template
- Header/body/hyperlink/button styles moved out of shiny css template as they are in the main template
- nav-pills and nav-tabs css added to main css template
- flexdashboard css added to shiny css template at the bottom (Component: small-box and varBox colours hack)

### CODE REVIEW

File: inst/text/shiny/shiny_css.css [hunk 1]
```diff
@@ -143,3 +92,109 @@ th, td {
 }
 
 /*
+
+
+/* CSS styles used for flexdashboard */
```
Comment (RosalynLP) [inline inst/text/shiny/shiny_css.css:97]: I think this is maybe misleading comment as it implies these styles are only used on flexdashboards when in fact they are flexdashboard-like styles being lifted for use elsewhere. 
```suggestion
/* CSS styles from flexdashboard */
```

**Review by RosalynLP — APPROVED** [review]
I'm guessing this addresses issue #66 
If so I'm happy with the changes here - looks good!

**Review by rubenv95 — APPROVED** [review]
It's looking good and works as expected! I did a little bit of playing about with it, I have suggested a couple of changes for stylistic reasons. 

Also I might suggest removing the underline from the active TOC and maybe using phs-purple-50? Again, completely subjective stylistic opinion so feel free to ignore. 

I've only tested an Rmarkdown, haven't test the shiny.

File: inst/text/phs_style.css [hunk 2]
```diff
@@ -61,3 +87,48 @@
...
+.nav-pills>li>a {
+    border-radius: 4px;
+    color: var(--phs-purple);
+}
+
+/* Navtabs - one tab with two+ indicators */
+.nav-tabs {
+    float: left;
+    margin-bottom: -1px;
+    color: var(--phs-purple);
```
[the remark this answers is not in the document]
Comment (alan-y) [reply inst/text/phs_style.css:122]: @rubenv95 - thanks, I've implemented your suggestions for TOC and .nav-tabs and just needs some testing to make sure all is good. I checked TOC already but might need you to help with checking .nav-tabs please

**Comment by alan-y** [comment]
> I'm guessing this addresses issue #66 If so I'm happy with the changes here - looks good!

Thanks @RosalynLP - yeah it partly addresses that issue but I still need to finish the PHS HTML template before I close that issue off (it's in another PR)

**Review by rubenv95 — APPROVED** [review]
As discussed over Teams, I have made a couple minor tweaks to the style to nav-tabs. I properly tested my initial suggestions and found a few ways to improve it. 
Also I did a little bit of consistency tweaking of the formatting. Happy for it to merge if you are happy with my changes!

**Pull request merged**